drafts/bitcartel-nu0-tx-format/draft.rst

```diff
@@ -0,0 +1,207 @@
+* provide replay protection between Legacy and Overwinter network upgrade.
+* provide replay protection between different branches post-Overwinter e.g. Overwinter and Sapling upgrade.
+* enable a branch to support multiple transaction version formats.
+* ensure transaction formats are parsed uniquely across parallel branches.
+* support transaction expiry as specified in the Transaction Expiry [#zip-0???]_.
+
+Specification
+=============
+
+Zcash launched with support for upstream Bitcoin version 1 transactions and a custom version 2 transaction format which added fields required for shielded transactions.
+
+======== =============== =========================== =======
+Version  Field           Description                 Type
+======== =============== =========================== =======
+>= 1     version         positive value              int32
+>= 1     in_count        varint                      1-9 bytes
+>= 1     tx_inputs       list of inputs              vector
+>= 1     out_count       varint                      1-9 bytes
+>= 1     tx_outputs      list of outputs             vector
+>= 1     lock_time       block height or timestamp   uint32
+>= 2     nJoinSplit      varint                      1-9 bytes
+>= 2     vJoinSplit      list of joinsplits          vector
+>= 2     joinSplitPubKey joinSplitSig public key     32 bytes
+>= 2     joinSplitSig    signature                   64 bytes
+======== =============== =========================== =======
+
+Version 1 and 2 transaction formats are legacy formats.
+
+Version 3 transaction format will be introduced for Overwinter.
+
+The version field must have its most significant bit set.
+
+There will be new fields for:
+
+* branch id
+* expiry height
+
+======== =============== =========================== =======
+Version  Field           Description                 Type
+======== =============== =========================== =======
+>= 3     version         must have MSB set           int32
+>= 3     branch_id       branch/fork identifier      uint32
+>= 3     expiry_height   block height                uint32
+>= 1     in_count        varint                      1-9 bytes
+>= 1     tx_inputs       list of inputs              vector
+>= 1     out_count       varint                      1-9 bytes
+>= 1     tx_outputs      list of outputs             vector
+>= 1     lock_time       block height or timestamp   uint32
+>= 2     nJoinSplit      varint                      1-9 bytes
+>= 2     vJoinSplit      list of joinsplits          vector
+>= 2     joinSplitPubKey joinSplitSig public key     32 bytes
+>= 2     joinSplitSig    signature                   64 bytes
+======== =============== =========================== =======
+
+
+Transaction Version
+-------------------
+
+The version field is always serialized in little-endian format.
+
+Version 1 transaction 5c6ba844e1ca1c8083cd53e29971bd82f1f9eea1f86c1763a22dd4ca183ae061
+
+* begins with 0x01000000
+* 32-bit signed integer 00 00 00 01 == 1
```

Comment:
This has the opposite problem (an integer that looks like a byte sequence). Just say "32-bit signed integer 1".